### ISSUE TITLE
alpine docker image is missing udhcpc script

### ISSUE THREAD

**Issue by larsks** [alpinelinux/docker-alpine#61]
I'm using Alpine's Docker image with Podman to test out [OVN][] support for DHCP. I'm spawning a container against an existing network namespace and then trying to use `udhcpc` to configure the interface(s).

[podman]: https://podman.readthedocs.io/en/latest/
[ovn]: http://www.openvswitch.org/support/dist-docs/ovn-architecture.7.html

It turns out that while the Alpine image includes busybox `udhcpc`, it does not include the `/usr/share/udhcpc/default.script` script. This means that while it will happily acquire a DHCP lease, it won't actually configure interfaces.

**Comment by oxr463** [comment]
It should be there, see the package contents here, <https://pkgs.alpinelinux.org/contents?file=&path=&name=busybox&branch=edge&repo=main&arch=x86_64>.

Also running locally returns the following:

```sh
docker run -it alpine:edge
/ # ls /usr/share/udhcpc/
default.script
```

**Comment by larsks** [comment]
Yes, it looks like someone must have noticed. Given the "last updated" date on the `3.11` tag it's possible it was even due to this report...

```
$ for version in 3.8 3.9 3.10 3.11 edge; do 
> echo -n "$version: "
> docker run -it --rm alpine:$version ls -l /usr/share/udhcpc 2>&1
> done
3.8: ls: /usr/share/udhcpc: No such file or directory
3.9: ls: /usr/share/udhcpc: No such file or directory
3.10: ls: /usr/share/udhcpc: No such file or directory
3.11: total 4
-rwxr-xr-x    1 root     root          3104 Dec 18 18:04 default.script
edge: total 4
-rwxr-xr-x    1 root     root          3104 Dec 18 18:04 default.script
```

It would be awesome if we could get that fix back into the pre-3.11 images.

**Comment by ncopa** [comment]
it was fixed due to this report: https://gitlab.alpinelinux.org/alpine/aports/issues/10586

Fixed in this commit: alpinelinux/aports@d8d56f3702cefbfb5433283d1a5ac53e0884198c

I'm not sure we want backport it to older alpine, so we dont break things for people who have workarounds for this which depends on the broken behavior.

**Comment by ncopa** [comment]
I'm closing this since its fixed in `alpine:latest` and `alpine:edge`. Nothing more for us to do.
